omero-ssh-systemd/Dockerfile

```diff
@@ -14,10 +14,19 @@ RUN sed -i \
 	's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' \
 	/etc/pam.d/sshd
 
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
+
 RUN useradd omero && \
 	echo 'omero:omero' | chpasswd && \
 	echo "omero ALL= (ALL) NOPASSWD: ALL" >> /etc/sudoers.d/omero
 
+RUN rm -f /lib/systemd/system/systemd*udev* ; \
+	rm -f /lib/systemd/system/getty.target;
+
+RUN yum install -y initscripts \
```

Comment:
https://github.com/ome/omero-install/blob/develop/linux/test/test_services.sh meant to be usable across other platforms, it cost nothing to just add `RUN yum install -y initscripts` into centos7 Dockerfiles